pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -577,6 +577,11 @@ func init() {
 					Verbs:     sets.NewString("get", "create", "delete"),
 					Resources: sets.NewString("pods"),
 				},
+				// RecycleVolumeByWatchingPodUntilCompletion
+				{
+					Verbs:     sets.NewString("watch"),
```

Comment:
OK, I will add list as well. Seems to work without it because it's just watching recycler pod events in order to forward them to the PV being recycled. The actual check for whether the recycler has finished is still done by watching the recycler pod.

Yeah it needs write permissions, it already had them.